### DISCUSSION TITLE
[Portal Engine Bundle] Tag navigation doesn't work without Folder navigation

### DISCUSSION THREAD

**Discussion by Maksonis** [pimcore/pimcore#11955] [Q&A / Support] [answered]
When I select only Tag Navigation, without Folder navigation,

![image](https://user-images.githubusercontent.com/17277953/165062782-340bab0d-683c-4ce5-8be1-3951e338546a.png)

Tags are shown in frontend, but filter doesn't work:

![image](https://user-images.githubusercontent.com/17277953/165062986-8f1af36b-80ce-46ba-9c35-e00ae4f8734a.png)

With Folder navigation it works:

![image](https://user-images.githubusercontent.com/17277953/165063072-1584a445-3342-4b5c-b6e3-7299df4389b2.png)

Side question: Is there a possibility to Consider child tags too:

![image](https://user-images.githubusercontent.com/17277953/165063265-53c4e891-5c07-4ae6-9053-58ab1534a045.png)

?

**Comment by fashxp** [comment, marked as answer]
will be fixed in next bugfix release, ETA Wednesday May 11th
